spec/requests/errors_spec.rb

```diff
@@ -25,7 +25,7 @@
   describe '500 internal server error' do
     subject { -> { patch organization_message_url(organization, message, as: user) } }
 
-    let(:message) { create(:message, creator_id: user.id) }
+    let(:message) { create(:message, creator_id: user.id, organization: organization) }
```

Comment:
I didn't know that you could create a resource by passing in the association like this if you delegate to another resource.
I knew you could use the delegate to look it up, but didn't realize you could also create it like this. Cool :sunglasses: